frontend/src/main/pages/Moderate.js

```diff
@@ -1,24 +1,40 @@
 import React from "react";
```

Comment:
Filename should have been `ModeratePage.js` 

[@Tiffany Gu](https://ucsb-cs156-s25.slack.com/team/U08KMMXCBRP)
 (and 

Otherwise, this breaks the naming convention we’ve used throughout all of the React apps in this course, namely that the filenames for Page components end in Page.js    I really should make you go back and change it, but if that’s the only thing I find, I’m going to let it go, because we just need to wrap things up.